lib/gui/app/components/wifi/react/keyboard/index.jsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2018 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const Types = require('prop-types')
+const React = require('react')
+const Keyboard = require('react-virtual-keyboard')
+
+class KBInput extends React.PureComponent {
+  componentDidMount () {
+    this.keyboard.interface.keyaction.enter = (base) => {
+      return this.keyboard.interface.keyaction.accept(base)
+    }
+  }
+
+  render () {
+    console.log('value', this.props.value, typeof this.props.value)
+    console.log('type', this.props.type, typeof this.props.type)
```

Comment:
Leftover debug statements? If you'd like to keep them around, I'd suggest using the `debug` module (see elsewhere in Etcher)

Reply:
Yep, it's still a WIP so I'm gonna sanitize all console.logs and maybe introduce a bit of analytics too